components/questions.js

```diff
@@ -44,7 +45,16 @@ export default function Questions(props) {
 
   function handleClick(next) {
     if (next && confidence === DEFAULT_CONFIDENCE) {
-      setError(true);
+      setError("error1");
+    }
+    // if it is the last question and the user hasn't typed at least one comment then set error when the user clicks next button
+    else if (
+      next &&
+      props.counter == props.total &&
+      props.count == 0 &&
+      comment.length < MIN_LENGTH
+    ) {
+      setError("error2");
```

Comment:
Please use more more descriptive names so that the code is easier to maintain, for example here `noComment` or `missingComment`